dandi/metadata.py

```diff
@@ -455,11 +455,22 @@ def extract_sex(metadata: dict) -> Optional[models.SexType]:
 def extract_strain(metadata: dict) -> Optional[models.StrainType]:
     value = metadata.get("strain", None)
     if value is not None and value != "":
```

Comment:
while at it, could be simplified similarly to below
```suggestion
    if value:
```